test/testdata/resolver/final_module.rb

```diff
@@ -0,0 +1,58 @@
+# typed: true
+# disable-fast-path: true
```

Comment:
Disabling fast path here is a regression. As currently implemented, the "redeclared as final" errors are not incompatible with how LSP / fast path works.

Reply:
Not sure what you mean. Should I remove this comment? (If I do, the tests about redundant modifiers fail.) Should we not test for redundant modifiers?

Reply:
Resolved offline: It's tricky to test for redundant modifiers statically with LSP. Let's not check for them.